traits/trait_types.py

```diff
@@ -1913,11 +1914,14 @@ class BaseEnum(TraitType):
         The enumeration of all legal values for the trait.  The expected
         signatures are either:
 
-        - a single list, enum.Enum or tuple.  The default value is the first
-          item in the collection.
+        - a single list, enum.Enum, tuple or a collection.  The default value
+          is the first item in the collection. The collection should conform to
+          the collections.abc.Collection interface. That is, it at least
+          provides the __contains__, len and __iter__ methods.
         - a single default value, combined with the values keyword
           argument.
-        - a default value, followed by a single list enum.Enum or tuple.
+        - a default value, followed by a single list enum.Enum, tuple or
+          collection conforming to collections.abc.Collection
```

Comment:
There's a paragraph below this than ends prematurely with `otherwise it is the first item stored in the .`.  While we're fixing up the docstring, should we fix that sentence, too?

Reply:
Fixed it. Thanks